tests/UnitTests/CoreGraphics.drawing/EntryPoint.cpp

```diff
@@ -96,29 +90,93 @@ std::shared_ptr<DrawingTestConfigImpl> _configImpl;
 }
 
 #ifdef WIN32
-#include <COMIncludes.h>
+#include <roapi.h>
+#include <WexTestClass.h>
+#include <wrl\client.h>
 #include <wrl\wrappers\corewrappers.h>
-#include <COMIncludes_end.h>
-using namespace Microsoft::WRL::Wrappers;
+#include <windows.storage.h>
+
+using namespace ABI::Windows::Storage;
+using namespace Microsoft::WRL;
+using namespace Windows::Foundation;
+
+Wrappers::HString GetAppDataPath() {
+    Wrappers::HString toReturn;
+    ComPtr<IStorageFolder> folder;
+    ComPtr<IApplicationDataStatics> applicationDataStatics;
+    ComPtr<IApplicationData> applicationData;
+    ComPtr<IStorageItem> storageItem;
+
+    if (FAILED(Windows::Foundation::GetActivationFactory(Wrappers::HStringReference(RuntimeClass_Windows_Storage_ApplicationData).Get(),
+                                                         &applicationDataStatics))) {
+        return toReturn;
+    }
+
+    if (FAILED(applicationDataStatics->get_Current(&applicationData))) {
+        return toReturn;
+    }
+
+    if (FAILED(applicationData->get_LocalFolder(&folder))) {
+        return toReturn;
+    }
+
+    if (FAILED(folder.As<IStorageItem>(&storageItem))) {
+        return toReturn;
+    }
+
+    if (FAILED(storageItem->get_Path(toReturn.GetAddressOf()))) {
+        return toReturn;
+    }
+
+    return toReturn;
+}
 #endif
 
-int main(int argc, char** argv) {
 #ifdef WIN32
-    // Initialize the windows runtime, with uninitialized upon destructor invocation.
-    RoInitializeWrapper initialize(RO_INIT_MULTITHREADED);
-    if (FAILED(initialize)) {
-        return -1;
-    }
+BEGIN_MODULE()
+MODULE_PROPERTY(L"RunAs", L"UAP")
+END_MODULE()
+
+MODULE_SETUP(ModuleSetup) {
+#else
+int main(int argc, char** argv) {
 #endif
+
 #ifdef WINOBJC
     // NOTE: Current implementation of CoreGraphics requires UIKit to be loaded
     //      Remove once CoreGraphics has been updated
     [UIView class];
 #endif // WINOBJC
+
+#ifdef WIN32
+    if (FAILED(RoInitialize(RO_INIT_MULTITHREADED))) {
+        return FALSE;
+    }
+
+    Wrappers::HString path = GetAppDataPath();
+    if (path.IsValid()) {
+        unsigned int rawLength;
+        _wchdir(WindowsGetStringRawBuffer(path.Get(), &rawLength));
+    }
+
+    int argc = 1;
+    char* argv[] = { "UnitTests" };
+#endif
     testing::InitGoogleTest(&argc, argv);
 
     _configImpl = std::move(std::make_shared<CommandLineDrawingTestConfigImpl>(argc, argv));
```

Comment:
no. it can't. this is me stamping out argc/v to interop with the below call. See my comment on runtime params.